NOAA-EMC/EVS
# feature/update_mailto

<b>Note to developers: You must use this PR template!</b>

## Description of Changes

This updates `MAILTO` to reflect the current state of EVS developers and project leads.

Closes #542.

## Developer Questions and Checklist
* Is this a high priorty PR? If so, why and is there a date it needs to be merged by?
> No
* Do you have any planned upcoming annual leave/PTO?
> Full day 9/13, half day afternoon 9/16, and full day 9/20
* Are there any changes needed for when the jobs are supposed to run?
> N/A
  
- [ ] Jobs contain the approriate file checking and don't run METplus for any missing data.

## Testing Instructions

Not sure if we want to test anything with this. We could test doing fake path for something to see the emails work, but I don't want to send anyone emails unnecessarily about missing data when it is fake. @AndrewBenjamin-NOAA is also on leave and I don't know if we want to clog up his inbox more. We could also wait till he returns to do a test to make sure he gets the emails.
